packages/jest-cli/src/cli/index.js

```diff
@@ -67,6 +68,15 @@ const runCLI = async (
       outputStream,
     );
 
+    if (argv.clearCache) {
+      configs.map(config => {
```

Comment:
`forEach` over `map`. (You ignore the return value anyways)

Reply:
I usually prefer map to prevent accidental mutation. I suppose here it doesn't matter as the process is about to end. I will defer to you on this.